src/ziggurat/streams.clj

```diff
@@ -5,9 +5,10 @@
             [ziggurat.metrics :as metrics]
             [ziggurat.config :refer [ziggurat-config]]
             [sentry-clj.async :as sentry]
-            [ziggurat.mapper :as mpr]
             [ziggurat.channel :as chl]
-            [ziggurat.kafka-delay :as kafka-delay]
+            [ziggurat.map :as zmap]
```

Comment:
everywhere else this namespace is referred to as `umap`. Why `zmap` here?

Reply:
done